upgrade/1.5.1/README.md

```diff
@@ -0,0 +1,312 @@
+
+## Bug fixes and improvements
+
+* Updated [IMS](../../glossary.md#image-management-service-ims) to allow for remote node builds
+* Support for Paradise hardware
+* Fixed an issue where `hms-discovery` would put default credentials into vault disabling non default credentials
+* Fixed an issue where `cray-ipxe` generated scripts would cause boot errors
+* Fixed an issue where [BOS](../../glossary.md#boot-orchestration-service-bos) v2 would send large queries to [CFS](../../glossary.md#configuration-framework-service-cfs) resulting 503 and 431 responses
+* Fixed an issue where [PCS](../../glossary.md#power-control-service-pcs) was adding invalid components to power operations
+* Fixed an issue where [IUF](../../glossary.md#install-and-upgrade-framework-iuf) would encounter a race condition and stall on transitioning to new stages
+* Fixed an issue where `Thanos` service is configured without storage limits
+* Fixed an issue where `cray-dns-unbound-manager` `stderr` handling can corrupt configuration
+* Fixed an issue where [SAT](../../glossary.md#system-admin-toolkit-sat) `status` unnecessarily queries BOS for session template for every component
+* Fixed an issue where a PATCH to a BOS v2 session to change its name results in a bad state
+* Updated `cray-hms-rts-init` job to include a TTL
+* Updated `node-exporter` configuration to monitor SNMP counters
+* Updated documentation to cover switch configuration for [NCNs](../../glossary.md#non-compute-node-ncn)
+* Fixed an issue where a PATCH call to BOS v2 components with a filter and non existent component ID would result in 503 and 431 errors
+* Fixed an issue where BOS operators would output errors when all nodes exceed retry limit
+* Fixed an issue where `cray-upload-recovery-images` fails to upload recovery firmware
+* Fixed an issue where `cm health report slingshot refresh` command was giving a trace back error
+* Updated documentation for IUF to include rolling reboots after upgrading [HSN](../../glossary.md#high-speed-network-hsn) NIC firmware
+* Updated documentation to provide instructions for creating a new Nexus repository and adding RPMs
+* Updated documentation for [CAPMC](../../glossary.md#cray-advanced-platform-monitoring-and-control-capmc) to warn about URL character limits
+* Fixed an issue where BOS v2 requests for too many nodes from PCS exceeding the URL character limit
+* Updated Nexus export and restore script to add better checks for determining existence of `nexus-bak` PVC or `nexus-backup` job
+* Updated documentation for `fix_failed_to_start_etcd_on_master` to better specify how to add a process
+* Fixed an issue with IUF where process-media/pre-install-check dislikes PDF file
+* Fixed an issue where `backup_smd_postgres.sh` script is not executable
+* `CAST-34869`: Fixed an issue where `cray-hms-rts-snmp` would enter encounter a segmentation fault
+* `CAST-34141`: Fixed an issue where the `QLogic` driver would crash the system
+
```

Comment:
Add a note about fix for spire-agent version issue fixed with hotfix `CASMTRIAGE-6715-noos-repo`
```suggestion
* Fixed an issue where an old version of the spire-agent was loaded into Nexus resulting in failed ARM compute builds

```